include/client/dbif.h

```diff
@@ -138,7 +138,7 @@ class NCWrapper : public QObject {
       std::string name);
   dbif::MethodResultPromise* handleSetCommentRequest(data::NodeID id,
       std::string comment);
-  dbif::MethodResultPromise* handleChangeDataRequest();
+  dbif::MethodResultPromise* handleChangeDataRequest(data::NodeID id, QSharedPointer<dbif::ChangeDataRequest> change_data_request);
```

Comment:
and it is temporary code, when new model will be implemented all this things will be removed